components/html-block/demo/html-block.html

```diff
@@ -9,6 +9,60 @@
 		<script type="module">
 			import '../../demo/demo-page.js';
 			import '../html-block.js';
+			import { provideInstance } from '../../../mixins/provider-mixin.js';
+
+			let profileCardAccess;
+			const hasProfileCardAccess = () => {
+				if (profileCardAccess) return profileCardAccess;
+
+				profileCardAccess = new Promise(resolve => {
+
+					resolve(true); // temp
+
+					if (!D2L.LP) {
+						resolve(false);
+						return;
+					}
+
+					D2L.LP.Web.UI.Rpc.Connect(
+						D2L.LP.Web.UI.Rpc.Verbs.GET,
+						new D2L.LP.Web.Http.UrlLocation('/d2l/lp/profilecardsaccess/AccessCheck'),
+						null,
+						{ success: response => resolve(response) }
+					);
+				});
+
+				return profileCardAccess;
+			};
+
+			provideInstance(document, 'html-block-renderers', [{
+				prepare: async() => {
+					return hasProfileCardAccess();
+				},
+				render: elem => {
+
+					const mentions = elem.querySelectorAll('[data-mentions-id]');
+					if (mentions.length === 0) return elem;
+
+					mentions.forEach(mention => {
+
+						const userId = mention.getAttribute('data-mentions-id');
+						if (!userId) return;
+
+						const anchor = document.createElement('a');
```

Comment:
This is currently what is rendered in Spark, but in theory we can render a fancy user profile card.